drivers/sensor/lsm6dsl/lsm6dsl_spi.c

```diff
@@ -60,8 +70,9 @@ static int lsm6dsl_raw_read(struct spi_config *spi_cfg, u8_t reg_addr,
 static int lsm6dsl_raw_write(struct spi_config *spi_cfg, u8_t reg_addr,
```

Comment:
Should be:
static int lsm6dsl_raw_write(struct lsm6dsl_data *data, u8_t reg_addr,